js/client_exporter/excel_creator.js

```diff
@@ -26,6 +26,9 @@ var Class = require("../core/class"),
     STYLE_FILE_NAME = "styles.xml",
     WORKSHEETS_FOLDER = "worksheets",
     WORKSHEET_FILE_NAME = "sheet1.xml",
+    WORKSHEET_HEADER_XML = "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"yes\"?><worksheet xmlns=\"http://schemas.openxmlformats.org/spreadsheetml/2006/main\" " +
+        "xmlns:r=\"http://schemas.openxmlformats.org/officeDocument/2006/relationships\" xmlns:mc=\"http://schemas.openxmlformats.org/markup-compatibility/2006\" " +
+        "mc:Ignorable=\"x14ac\" xmlns:x14ac=\"http://schemas.microsoft.com/office/spreadsheetml/2009/9/ac\">",
```

Comment:
I recommend to convert the file to ES6 and use [String Interpolation](http://es6-features.org/#StringInterpolation)

Reply:
Yes, the '[[ ..].join("")]' expression in several lines of code doesn't seems to be easy to read.
However, I suggest to change it a bit later, after our tests will create an impression of a good coverage.
For now, I want to export this string as a 'const' to avoid duplicates in tests.
Defer?

Reply:
It's OK if it will not be postponed too far :)